### PR TITLE
Run tests against rails 6.0.0.beta3

### LINKED ISSUE #118: Add tests against Rails edge?
As development for Rails 6.0 in underway, maybe we should add Rails edge as a target for our tests?

### PR DESCRIPTION
**Pull request by Spone**
closes #118

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ gemfile:
   - gemfiles/rails_5.0.x.gemfile
   - gemfiles/rails_5.1.x.gemfile
   - gemfiles/rails_5.2.x.gemfile
+  - gemfiles/rails_6.0.x.gemfile
```
Comment (Spone) [inline .travis.yml]: We forgot to add this last time so the tests weren't actually run for Rails 6

**Comment by Spone** [comment]
I had to drop support for Ruby 2.3 and 2.4 because ActiveSupport 6.0.0.beta3 requires Ruby 2.5

**Comment by Spone** [comment]
Closed in favor of #134 that includes both these commits and fixes the tests.
